.php-cs-fixer.dist.php

```diff
@@ -37,7 +37,10 @@
         '@PHPUnit100Migration:risky' => true,
         '@PhpCsFixer' => true,
         '@PhpCsFixer:risky' => true,
-        'general_phpdoc_annotation_remove' => ['annotations' => ['expectedDeprecation']], // one should use PHPUnit built-in method instead
+        'general_phpdoc_annotation_remove' => ['annotations' => [
+            'expectedDeprecation', // one should use PHPUnit built-in method instead
+            'group',
```

Comment:
as we already have some reasons here for the non-obvious removal of annotation, would be good to continue with it for new annotation, so when JohnDoe open this file in future, they do know the reason.